docs/custom-spans.asciidoc

```diff
@@ -7,14 +7,14 @@ endif::[]
 
 == Custom Spans
 
-If you want to track and time a custom event that happens in your application during a transaction,
-you can add a new span to an existing transaction.
-
 This is an example of how to use custom spans.
 For general information about the Elastic APM Node.js Span API,
 see the <<span-api,Span API documentation>>.
 
-In the example below we create an Express app that times how long it takes to:
+If you want to track and time a custom event that happens in your application during a transaction,
+you can add a new span to an existing transaction.
+
+In the example below, we create an Express app that times how long it takes to:
```

Comment:
Any particular reason for rearranging these paragraphs?

Reply:
I was trying to match the format used in `custom-transactions.asciidoc`. That page opens with a similar sentence, so it made sense to me to start both pages in the same manner. 
<img width="639" alt="screen shot 2018-10-24 at 12 53 55 pm" src="https://user-images.githubusercontent.com/5618806/47457479-13a87300-d78c-11e8-821d-cfaaee3136af.png">